src/standalone/Vegetation/Canopy.jl

```diff
@@ -140,6 +148,7 @@ function CanopyModel{FT}(;
     photosynthesis::AbstractPhotosynthesisModel{FT},
     conductance::AbstractStomatalConductanceModel{FT},
     hydraulics::AbstractPlantHydraulicsModel{FT},
+    energy = PrescribedCanopyTempModel{FT}(),
```

Comment:
Setting this to be the default so we dont need to change all of other files right now. We can decide later if there should be a default and what it should be.